### DISCUSSION TITLE
V2.0 official release

### DISCUSSION THREAD

**Discussion by akvsx** [Akshay-Vs/PassLock#22] [Announcements]
<h1 align="center" ><b>2.0 Release notes</b></h1>

## Looks

- New terminal interface
- Smooth transition between window
- Improved Console graphics
- Themes

## Security

- New encoder
-  Two-step verification
- AES 256
- Parallel Encryption

## I/O

- Automatic file locator
- JSON files to store password data
- Improved Path_dir

## Performance

- Improved Startup time
- Fast Encoder
- Improved Stability
- Optimised CPU usage

## Exception handling

- Removed "Path Interrupted Error", thus it is now handled completely automatically 

## Other Features

- Switch between users
- Change the Private key without restarting the application
- View recent activities
- Editing Passwords
- Notifications

##  Commands

- `new` to create a new password
- `show <id name>`  to show saved password
- `--del <id name>`  to delete a password
- `--Exit` to exit the application
- `--list` to list all saved password
- `-p`  to switch to another  folder
- `restart  with <private key>` to restart passlock with another private key without login
- `--move` to move passwords to another folder
- `-c` to copy passwords to another folder
- `edit` to edit passwords (Only available after viewing password)
- `-r <number of charectors>` to generate a random password when creating or editing a password

<hr /><em>This discussion was created from the release <a href='https://github.com/Akshay-Vs/PassLock/releases/tag/v2.0'>V2.0 official release</a>.</em>

**Comment by Codder-Girl** [comment]
That terminal interface looks awesome

**Comment by codey-ash** [comment]
That's a big update
